fabfile.py

```diff
@@ -33,10 +31,22 @@
     subscribe,
     unsubscribe,
     update_basic_packages,
-    update_packages,
     upstream_install,
     vm_create,
     vm_destroy,
     vm_list,
     vm_list_base,
 )  # flake8: noqa
+from automation_tools.repository import (
+    create_custom_repos,
+    delete_custom_repos,
+    disable_beaker_repos,
+    disable_repos,
+    enable_repos,
+    enable_satellite_repos,
+    manage_custom_repos,
+)  # flake8: noqa
+from automation_tools.utils import (
+    distro_info,
+    update_packages
+)  # flake8: noqa
```

Comment:
Actually I will have to add `# noqa` for every line. I removed all noqa comments and flake8 shows the following:

```
./fabfile.py:2:1: F401 'add_repo' imported but unused
./fabfile.py:2:1: F401 'cdn_install' imported but unused
./fabfile.py:2:1: F401 'clean_rhsm' imported but unused
./fabfile.py:2:1: F401 'client_registration_test' imported but unused
./fabfile.py:2:1: F401 'create_personal_git_repo' imported but unused
./fabfile.py:2:1: F401 'downstream_install' imported but unused
./fabfile.py:2:1: F401 'errata_upgrade' imported but unused
./fabfile.py:2:1: F401 'fix_hostname' imported but unused
./fabfile.py:2:1: F401 'foreman_debug' imported but unused
./fabfile.py:2:1: F401 'install_errata' imported but unused
./fabfile.py:2:1: F401 'install_katello_agent' imported but unused
./fabfile.py:2:1: F401 'install_prerequisites' imported but unused
./fabfile.py:2:1: F401 'iso_download' imported but unused
./fabfile.py:2:1: F401 'iso_install' imported but unused
./fabfile.py:2:1: F401 'partition_disk' imported but unused
./fabfile.py:2:1: F401 'performance_tuning' imported but unused
./fabfile.py:2:1: F401 'product_install' imported but unused
./fabfile.py:2:1: F401 'remove_katello_agent' imported but unused
./fabfile.py:2:1: F401 'run_errata' imported but unused
./fabfile.py:2:1: F401 'setup_abrt' imported but unused
./fabfile.py:2:1: F401 'setup_ddns' imported but unused
./fabfile.py:2:1: F401 'setup_default_capsule' imported but unused
./fabfile.py:2:1: F401 'setup_default_docker' imported but unused
./fabfile.py:2:1: F401 'setup_email_notification' imported but unused
./fabfile.py:2:1: F401 'setup_fake_manifest_certificate' imported but unused
./fabfile.py:2:1: F401 'setup_firewall' imported but unused
./fabfile.py:2:1: F401 'setup_proxy' imported but unused
./fabfile.py:2:1: F401 'setup_vm_provisioning' imported but unused
./fabfile.py:2:1: F401 'subscribe' imported but unused
./fabfile.py:2:1: F401 'unsubscribe' imported but unused
./fabfile.py:2:1: F401 'update_basic_packages' imported but unused
./fabfile.py:2:1: F401 'upstream_install' imported but unused
./fabfile.py:2:1: F401 'vm_create' imported but unused
./fabfile.py:2:1: F401 'vm_destroy' imported but unused
./fabfile.py:2:1: F401 'vm_list' imported but unused
./fabfile.py:2:1: F401 'vm_list_base' imported but unused
./fabfile.py:40:1: F401 'create_custom_repos' imported but unused
./fabfile.py:40:1: F401 'delete_custom_repos' imported but unused
./fabfile.py:40:1: F401 'disable_beaker_repos' imported but unused
./fabfile.py:40:1: F401 'disable_repos' imported but unused
./fabfile.py:40:1: F401 'enable_repos' imported but unused
./fabfile.py:40:1: F401 'enable_satellite_repos' imported but unused
./fabfile.py:40:1: F401 'manage_custom_repos' imported but unused
./fabfile.py:49:1: F401 'distro_info' imported but unused
./fabfile.py:49:1: F401 'update_packages' imported but unused
```

Will continue skipping the entire file as we don't have any other code than F401.